app/models/hubstats/pull_request.rb

```diff
@@ -58,10 +64,30 @@ def self.create_or_update(github_pull)
           end
       end
 
+      if user.team && user.team.id
+        pull_data[:team_id] = user.team.id
+      end
+
       return pull if pull.update_attributes(pull_data)
       Rails.logger.warn pull.errors.inspect
     end
 
+    # Public - Updates the team_id for all pull requests from the past year
+    #
+    # Returns - nothing
+    def self.update_teams_in_pulls
+      puts "Gathering pulls from past year"
+      pulls = created_in_past_year(Date.today - 365)
```

Comment:
The past year makes sense for our org, but is an arbitrary value for someone else.  This should probably be a argument passed in at runtime of the rake task or set in a config file.